ocpp/messages.py

```diff
@@ -9,7 +9,7 @@
 from typing import Callable, Dict, Union
 
 from jsonschema import Draft4Validator
-from jsonschema import _validators as SchemaValidators
+from jsonschema import _keywords as SchemaValidators
```

Comment:
#479 already address the `ImportError` when using the version of jsonschema 4.19.0 and up. 
But I still want to inform you that this fix would break for everyone using a version older than 4.19.0.